sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/transport/message/HeartbeatMessage.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.csp.sentinel.transport.message;
+
+import java.util.Map;
+import java.util.function.Supplier;
```

Comment:
Should we use `java.util.function.Supplier` or `com.alibaba.csp.sentinel.util.function.Supplier`?

Reply:
Since the target JDK version has been upgraded to 1.8, we could use `java.util.function.Supplier`.